app/models/notifier.rb

```diff
@@ -1,17 +1,12 @@
 require "slack-notifier"
+require "json"
 
 class Notifier
-  attr_reader :webhook_url
-
-  def initialize
-    @webhook_url = ENV["SLACK_WEBHOOK_URL"]
-  end
-
-  def notify(message)
-    notifier = Slack::Notifier.new webhook_url do
+  def notify(notification)
+    notifier = Slack::Notifier.new ENV["SLACK_WEBHOOK_URL"] do
       defaults channel: ENV["SLACK_CHANNEL"], username: ENV["SLACK_USERNAME"]
     end
 
-    notifier.ping message
+    notifier.ping JSON.parse(notification.to_json)
```

Comment:
Why is the notification turned into a JSON, but then parsed back into a Ruby data type?

Reply:
Whoops! Yeah, great spot - thanks @AshGDS - didn't mean to leave that in. Fixed up 👍